blocks/library/embed/index.js

```diff
@@ -29,15 +29,12 @@ const HOSTS_NO_PREVIEWS = [ 'facebook.com' ];
 function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, keywords = [] } ) {
 	return {
 		title,
-
-		description: __( 'The Embed block allows you to easily add videos, images, tweets, audio, and other content to your post or page.' ),
-
+		description: 'Embed' !== title ?
```

Comment:
I would rather pass block's `name` and perform this check based on it.

Reply:
@gziolo will give that a try

Reply:
Cool, thanks 👍

Reply:
I have another idea which is simpler: You can pass `description` for the base Embed block and compose it based on title for all other occurences:

```js
description: description || __( `Paste URLs from ${ title } to embed the content in this block` ),
```

Reply:
@gziolo this looks good compared to title or name safe & fewer code changes :)

have updated the PR